src/workloads/execution/UpdateLargeDocuments.yml

```diff
@@ -0,0 +1,83 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: >
+  This workload measures the performance of large document updates in a replica set. First, it
```

Comment:
Super pedantic but isn't ```the performance of updating large documents``` more accurate. The workload is updating a single field in large documents.